api-tests/platform/targets/tgt_dev_apis_stdc/target.cmake

```diff
@@ -44,10 +44,14 @@ if(${SUITE} STREQUAL "CRYPTO")
 	)
 endif()
 if(${SUITE} STREQUAL "PROTECTED_STORAGE")
-	message(FATAL_ERROR "Protected Storage not supported")
+  list(APPEND PAL_SRC_C_NSPE
+    ${PSA_ROOT_DIR}/platform/targets/${TARGET}/nspe/protected_storage/pal_protected_storage_intf.c
+    )
 endif()
 if(${SUITE} STREQUAL "INTERNAL_TRUSTED_STORAGE")
-	message(FATAL_ERROR "Internal Trusted Storage not support")
+  list(APPEND PAL_SRC_C_NSPE
```

Comment:
4 spaces guideline to be followed.

Reply:
This commit has been removed from this PR (#136) because the change (with spacing as per the guideline) is included in the related outstanding PR #137 submitted to this repo.